### ISSUE TITLE
job error: expected string or buffer

### ISSUE THREAD

**Issue by davidcsterratt** [tokland/pysheng#16]
I've tried using pysheng-gui on this URL: https://books.google.co.uk/books?id=uGahCAAAQBAJ&pg=PA54&&hl=en&sa=X&ei=pkRmVdngBMKBUfLjgcgN&redir_esc=y#v=onepage&f=false
The downloader gets to 29/197, and then gives ```job error: expected string or buffer``` in the log and, on the command line:
```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/pysheng/gui.py", line 186, in download_book
    image_url = re.sub("w=(\d+)", "w=" + str(width), image_url0)
  File "/usr/lib/python2.7/re.py", line 151, in sub
    return _compile(pattern, flags).sub(repl, string, count)
TypeError: expected string or buffer
```
I'm running Ubuntu 14.04 with python 2.7.6.

**Comment by tokland** [comment]
fixed, thx 2fce22da

**Comment by davidcsterratt** [comment]
Marvellous - thank-you very much.
